larq/conftest.py

```diff
@@ -1,10 +1,17 @@
 import pytest
 import tensorflow as tf
 from packaging import version
+from tensorflow.python.distribute import strategy_combinations
 from tensorflow.python.eager import context
 
 from larq import context as lq_context
 
+if version.parse(tf.__version__) >= version.parse("1.15"):
+    strategy_combinations.set_virtual_cpus_to_at_least(3)
```

Comment:
This sets the device config globally for all unittests since otherwise we would need to properly reset the execution context after each unittest.